app/waf/src/keys.ts

```diff
@@ -41,3 +43,20 @@ export namespace WafBindingKeys {
     );
   }
 }
+
+export function getTenantId(reqCxt: RequestContext): Promise<string> {
```

Comment:
I think this function did no extra work but `reqCxt.get(WafBindingKeys.Request.KeyTenantId);`
So I prefer to inline the function in monitor.controller.ts(with no try{}catch{} ..)